modules/ROOT/pages/microsoft/microsoft-365-ops-connector-reference.adoc

```diff
@@ -3,327 +3,317 @@ ifndef::env-site,env-github[]
-| Reconnection a| Reconnection |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
-| Consumer Key a| String |  The OAuth consumerKey as registered with the service provider |  |x
-| Consumer Secret a| String |  The OAuth consumerSecret as registered with the service provider |  |x
-| Authorization Url a| String |  The service provider's authorization endpoint URL. |  `+https://login.microsoftonline.com/{tenant}/oauth2/authorize+` |
-| Access Token Url a| String |  The service provider's accessToken endpoint URL |  `+https://login.microsoftonline.com/{tenant}/oauth2/token+` |
-| Scopes a| String |  The OAuth scopes to be requested during the dance. If not provided, the connector defaults to those in the annotation. |  |
-| Resource Owner Id a| String |  The resourceOwnerId which each component should use if it doesn't reference otherwise. |  |
-| Before a| String |  The name of a flow to execute right before starting the OAuth dance. |  |
-| After a| String |  The name of a flow to execute right after an accessToken has been received. |  |
-| Listener Config a| String |  A reference to a `<http:listener-config />` to use to create the listener to catch the access token callback endpoint. |  |x
-| Callback Path a| String |  The path of the access token callback endpoint. |  |x
-| Authorize Path a| String |  The path of the local HTTP connector that triggers the OAuth dance. |  |x
-| External Callback Url a| String |  If the callback endpoint is behind a proxy or should be accessed through a non-direct URL, use this parameter to tell the OAuth provider the URL to use to access the callback. |  |
-| Object Store a| String |  A reference to the object store to store each resource owner ID's data. If not specified, the runtime automatically provisions the default one. |  |
+| Scope a| String |  |  openid | 
+| Resource a| String |  |  | x
+| Response Type a| String |  |  query | 
+| Read Timeout a| Number |  Specifies the amount of time in milliseconds that the consumer waits for a response before it times out. Default value is 0, which means infinite. |  0 | 
+| Connection Timeout a| Number |  Specifies the amount of time in milliseconds that the consumer attempts to establish a connection before it times out. Default value is 0, which means infinite. |  0 | 
+| Host a| String |  Hostname of the proxy. If this property is not set, then no proxy is used, otherwise a proxy is used, but a proxy host must be specified. |  | 
+| Port a| Number |  Port of the proxy. If host is set then this property must be set and cannot be a negative number. |  | 
+| Username a| String |  Username used to authenticate against the proxy. If this property is not set, then no authentication is going to use against the proxy, otherwise this value must be specified. |  | 
```

Comment:
```suggestion
| Username a| String |  Username used to authenticate against the proxy. If this property is not set, then no authentication is used against the proxy; otherwise this value must be specified. |  | 
```